service/ExchangeService.js

```diff
@@ -263,6 +441,17 @@ var fetchFromBithumb= function(callback){
 
 };
 
+var addUsdPrice = function(arr){
```

Comment:
It's better to use (https://eslint.org/docs/rules/func-names) function statement
```js
function addUsdPrice(arr) {
...
```
or named function expression:
```js
var addUsdPrice = function addUsdPrice(arr) {
...
```